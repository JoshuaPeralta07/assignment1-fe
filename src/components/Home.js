import React from 'react';

function Home(props) {
    return (
        <div>
            <h1>Assignment 1</h1>
            <h2>Home page</h2>
        </div>
    );
}

export default Home;