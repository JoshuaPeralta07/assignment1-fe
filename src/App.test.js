import React from 'react';
import renderer from 'react-test-renderer';
import Home from "./components/Home";




test('renders Home and image correctly', () => {
  const homeComponent = renderer.create(<Home />);
  const imgComponent = renderer.create(<img src={require('./Images/dp.jpg')} alt="dp" />);

  const homeTree = homeComponent.toJSON();
  const imgTree = imgComponent.toJSON();

  expect(homeTree).toMatchSnapshot();
  expect(imgTree).toMatchSnapshot();
});