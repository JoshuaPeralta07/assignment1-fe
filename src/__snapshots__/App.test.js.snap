// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home and image correctly 1`] = `
<div>
  <h1>
    Welcome to MIS
  </h1>
  <img
    alt="dp"
    src="dp.jpg"
    style={
      Object {
        "border": 5,
        "borderColor": "black",
        "borderStyle": "solid",
        "display": "block",
        "margin": "auto",
        "paddingTop": 50,
        "width": "50%",
      }
    }
  />
</div>
`;

exports[`renders Home and image correctly 2`] = `
<img
  alt="dp"
  src="dp.jpg"
/>
`;

exports[`renders correctly 1`] = `
<div>
  <h1>
    Assignment 1
  </h1>
  <h2>
    Home page
  </h2>
  <img
    alt="dp"
    src="dp.jpg"
  />
</div>
`;
